xclim/data/fr.json

```
@@ -583,19 +582,19 @@
   },
   "RB_FLASHINESS_INDEX": {
     "long_name": "Indice Richards-Baker de torrentialité",
-    "description": "Indice Richards-Baker mesurant le caractère torrentiel d'un débit de rivière.",
+    "description": "{freq} de R-B indice, un indice mesurant le caractère torrentiel d'un débit de rivière.",
     "title": "Indice Richards-Baker de torrentialité",
     "abstract": "Mesure des oscillations du débit relatif au débit moyen, quantifiant la fréquence et la rapidité des changements de débits."
   },
   "FREQ_ANALYSIS": {
-    "long_name": "Période de retour {mode} {indexer} du flux de {window} jours",
-    "description": "Analyse de fréquence pour le {mode} {indexer} du flux de {window} jours, estimé avec la distribution {dist:f}.",
+    "long_name": "Période de retour du montant de débit",
+    "description": "Analyse de fréquence de débit du courant pour le {mode} {indexer} du flux de {window} jours, estimé avec la distribution {dist:f}.",
     "title": "Période de retour",
-    "abstract": "Analyse de fréquence selon un mode et une distribution."
+    "abstract": "Analyse de fréquence de débit du courant selon un mode et une distribution."
   },
   "STATS": {
-    "long_name": "{op} du {indexer} {freq:m} du flux journalier",
-    "description": "{op} du {indexer} {freq:m} du flux journalier.",
+    "long_name": "Statistique du flux quotidien",
+    "description": "{op} du {indexer} {freq:m} du flux quotidien.",
```

```suggestion
    "description": "{op} du {indexer} {freq:m} des débits quotidiens.",
```
Je trouve que ça sonne mal... Idées?

---

```suggestion
    "description": "{op} {freq:f} des débits quotidiens ({indexer}).",
```
?
Exemples : "Somme annuelle des débits quotidiens (été).", "Écart-type saisonnière des débits quotidiens (04-05 to 06-09)"

Le formatteur automatique atteint ses limites avec ce genre de cas... J'ai choisi "{freq:f}" parce les "op" les plus communes sont féminines : somme, moyenne.

---

Si tu pouvais aussi proposer un changement côté anglo puisque les deux soit synchronisé, ça serait cool.